launcher/ui/dialogs/ModDownloadDialog.h

```diff
@@ -66,6 +72,7 @@ private slots:
     PageContainer * m_container = nullptr;
     QDialogButtonBox * m_buttons = nullptr;
     QVBoxLayout *m_verticalLayout = nullptr;
+    ModPage *m_selected_page = nullptr;
```

Comment:
I would format this with clang-format, maybe format the 4 lines above too?

Reply:
Ah, I was just trying my best to get code to fit in instead of changing other things.